setup.py

```diff
@@ -107,7 +109,7 @@
             'visualization/style/*.woff',
             'visualization/*.html'
         ],
-        'openmdao.devtools.xdsm_viewer': [
+        'openmdao.visualization.xdsm_viewer': [
```

Comment:
looks like you missed updating the previous package  `devtools.problem_viewer` -> `visualization.n2_viewer`?